### PR TITLE
Dependabot/npm and yarn/del 7.1.0

### PR DESCRIPTION
**Pull request by oalders**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -18,12 +18,23 @@ const exp = require('posthtml-expressions');
 const include = require('posthtml-include');
 
 gulp.task('clean', async () => {
-  return del([
-    'dist/**/*',
-  ]);
+  const dir = 'dist';
```
Comment (mm-jpoole) [inline gulpfile.js:21]: Only using this once? Could either make use of it on line 28 or just get rid of it.

**Review by mm-jpoole — CHANGES_REQUESTED** [review]
Lookin' good! Sync makes sense here.

One slight nitpick,

File: gulpfile.js (hunk 1 above)
Comment (oalders) [reply gulpfile.js:21]: Nice catch!

**Review by mm-jpoole — APPROVED** [review]
:partying_face:

**Pull request merged**